azerothcore/azerothcore-wotlk
# fix(DB/formations): Adds formations to two groups in UBRS

## Changes Proposed:
- Adds formations for two separate Rage Talon Captains in UBRS. These two have fairly wide groupings, and it was possible to pull these groups in such a way that one group member wouldn't react. This PR adds everyone in each of these two groups to a formation for that group so they all react to aggro.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9409
- Closes https://github.com/chromiecraft/chromiecraft/issues/2425
- Closes https://github.com/chromiecraft/chromiecraft/issues/2426

## SOURCE:

Looked at a lot of UBRS videos, but didn't find any where the leftmost one was pulled to verify this was wrong. However, it seems plausible that a group like this would move together.

## Tests Performed:

- Ran SQL, no errors/warnings, expected number of rows added.
- Tested in-game:

https://user-images.githubusercontent.com/81782124/144711806-23c5918b-7801-4c47-a258-caa5b27061ab.mp4

## How to Test the Changes:


1. .go c 137959 will port you to an NPC just before the pull in question. If you aggro the leftmost NPC, the entire group should aggro.
2. For the second group, turn GM mode on so they don't aggro and .go c 137861. Clear the room a bit, turn off GM mode, and hiberate the dragon on the right. They should all aggro, including the left-most NPC.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
